spec/controllers/projects_controller_spec.rb

```diff
@@ -254,6 +254,43 @@
       before{ get :show, permalink: project.permalink, project_post_id: project_post.id, locale: :pt }
       it("should assign update to @update"){ expect(assigns(:post)).to eq(project_post) }
     end
+
+    context "@supporters" do
+      let(:project) { create(:project) }
+
+      before do
+        allow_any_instance_of(Project).to receive(:recurring?).and_return(true)
+      end
+
+      context "when there are paid subscriptions for the project" do
+        before do
+          user_1 = create(:user, name: 'fulano')
```

Comment:
Once you are creating users, what do you think about name the variables with the user's name and define them using `let`? 

```Ruby
let(:fulano) { create(:user, name: 'fulano') }
# ...

before do
  create(:subscription, :paid, project: project, user: fulano)
end 
```

And on the expectation, you could do like this:
```Ruby
expect(assigns(:supporters)).to contain_exactly(fulano, beltrano)
```